src/main.rs

```diff
@@ -288,13 +323,13 @@ fn service(config: Config) -> std::io::Result<()> {
         for controller in controllers.iter_mut() {
             let gpu_temp = controller.hw_mon.gpu_temp().unwrap_or_default();
 
-            let speed = config.speed_for_temp(gpu_temp);
-            if controller.hw_mon.fan_min() > speed || controller.hw_mon.fan_max() < speed {
+            let target_pwm = config.speed_for_temp(gpu_temp);
```

Comment:
This is enough to convert the percentage to PWM values:
```suggestion
            let speed = config.speed_for_temp(gpu_temp);
            let target_pwm = (speed as f32 * 2.55).round() as u32;
```

Reply:
1% would give 2,55 which is below 4 and would be ignored